scl/slack/slack.conf

```diff
@@ -45,6 +43,9 @@ block destination slack(
   use-system-cert-store(yes)
   ...)
 {
+
+@requires http "The slack() driver depends on the syslog-ng http module, please install the syslog-ng-mod-http (Debian & derivatives) or the syslog-ng-http (RHEL & co) package"
```

Comment:
The `@requires http` left on top of the file that defeats a purpose of this, please remove that.

Reply:
done.